src/screens/SearchScreen.js

```diff
@@ -0,0 +1,175 @@
+import React, {Component} from 'react';
```

Comment:
As I mentioned before, the `SearchScreen` is the actual page that you view, and the component that renders is dependent on the state. I had to do some weird stuff to get this to work... but it does. I'll try to explain below.